### ISSUE TITLE
21119827: Accessibility: Posting UIAccessibilityScreenChangedNotification doesn't work on app launch

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#1120]
#### Description

Summary:
When trying to set the initial accessibility item after the app launches per UIAccessibilityPostNotification, the item is not selected. Instead the first accessibility item in the hierarchy is selected.

In the view controller which is visible after launching:

```
override func viewDidAppear(animated: Bool) {
    super.viewDidAppear(animated)
    UIAccessibilityPostNotification(UIAccessibilityScreenChangedNotification, self.myButton)
}
```

This is how it is documented in “Moving the VoiceOver Cursor to a Specific Element“ Listing 9-1:
https://developer.apple.com/library/ios/featuredarticles/ViewControllerPGforiPhoneOS/Accessibility/AccessibilityfromtheViewControllersPerspective.html#//apple_ref/doc/uid/TP40007457-CH2-SW1

Steps to Reproduce:
- Enable VoiceOver
- Launch the sample app attached

Expected Results:
- Button in the center selected and read aloud

Actual Results:
- Title in the navigation bar selected and read aloud

Regression:

Notes:
It works as expected when the controller is displayed from another view controller.
## 

Product Version: 8.3.1
Created: 2015-05-27T10:16:12.844280
Originated: 2015-05-27T12:16:00
Open Radar Link: http://www.openradar.me/21119827
